runner.js

```diff
@@ -82,12 +82,10 @@ async function run_task(config, task) {
         clearTimeout(timeout);
 
         task.status = 'success';
-        if (config.verbose) {
-            output.log(
-                config,
-                `[task] Marked #${task._runner_task_id} (${task.name}) as success`
-            );
-        }
+        output.logVerbose(
+            config,
+            `[task] Marked #${task._runner_task_id} (${task.name}) as success`
```

Comment:
Wouldn't this compute the function arguments, e.g. construct the strings, every time?

Reply:
Doesn't matter for performance at all